src/query/pipeline/sinks/src/sync_sink.rs

```diff
@@ -108,6 +108,7 @@ impl<T: Sink + 'static> Processor for Sinker<T> {
         self.inner.interrupt()
     }
 
+    #[minitrace::trace(name = "Sinker::process")]
```

Comment:
Maybe move it to https://github.com/datafuselabs/databend/blob/main/src/query/pipeline/core/src/processors/processor.rs#L168 is better?